vxblastsms/tests/test_ussd.py

```diff
@@ -16,8 +14,10 @@ class TestBlastSMSUssdTransport(VumiTestCase):
 
     def setUp(self):
         request_defaults = {
-            'msisdn': '27729042520',
-            'provider': 'MTN',
+            'msisdn': '273334444',
+            'shortcode': '8864',
+            'sessionid': 'test_session_id',
+            'type': '2',  # resume
```

Comment:
It's probably worth making these real integers in issue #2 so you don't end up with comparison issues in future